<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryDialog</class>
 <widget class="QTabWidget" name="queryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>按时间查询</string>
   </attribute>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>样本信息</string>
    </property>
    <widget class="QTableView" name="tableView_TestResult">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>781</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>779</width>
       <height>184</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;Vemana2000&quot;;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#5555ff;&quot;&gt;样本号:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="edt_query_SampleNo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="ck_query_setting">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 85, 255);
font: 11pt &quot;Vemana2000&quot;;</string>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 85, 255);
font: 11pt &quot;Vemana2000&quot;;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#5555ff;&quot;&gt;开始时间&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDateTimeEdit" name="dt_query_Start">
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd h:mm AP</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 85, 255);
font: 11pt &quot;Vemana2000&quot;;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#5555ff;&quot;&gt;结束时间&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDateTimeEdit" name="dt_query_End">
          <property name="displayFormat">
           <string>yyyy-MM-dd h:mm AP</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="btn_query_date">
          <property name="font">
           <font>
            <family>WenQuanYi Zen Hei</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 170, 255);</string>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
          <property name="icon">
           <iconset resource="rc.qrc">
            <normaloff>:/image/Query.png</normaloff>:/image/Query.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tablevew_Patient">
        <property name="styleSheet">
         <string notr="true">alternate-background-color: rgb(255, 170, 0);
color: rgb(0, 0, 255);
font: 63 11pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="tab1">
   <attribute name="title">
    <string>按项目查询</string>
   </attribute>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>查询条件</string>
       </property>
       <widget class="QComboBox" name="cb_ItemName">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>26</y>
          <width>175</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>60</width>
          <height>27</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 85, 255);
font: 11pt &quot;Vemana2000&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#5555ff;&quot;&gt;项目名称&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dt_query_Start_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>175</width>
          <height>27</height>
         </rect>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd h:mm AP</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>70</y>
          <width>60</width>
          <height>27</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 85, 255);
font: 11pt &quot;Vemana2000&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#5555ff;&quot;&gt;开始时间&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dt_query_End_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>175</width>
          <height>31</height>
         </rect>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd h:mm AP</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>110</y>
          <width>60</width>
          <height>27</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 85, 255);
font: 11pt &quot;Vemana2000&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#5555ff;&quot;&gt;结束时间&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_ItemQuery">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>190</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>WenQuanYi Zen Hei</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 170, 255);</string>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
        <property name="icon">
         <iconset resource="rc.qrc">
          <normaloff>:/image/Query.png</normaloff>:/image/Query.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>样本信息</string>
       </property>
       <widget class="QTableView" name="tbView_SampleResult">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>381</width>
          <height>241</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">alternate-background-color: rgb(255, 170, 0);
color: rgb(0, 0, 255);
font: 63 11pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
